src/core/profiles/profiles.module.ts

```diff
@@ -5,11 +5,19 @@ import { UsersModule } from '../users/users.module';
 import { TypeOrmModule } from '@nestjs/typeorm';
 import { Skill } from '../skills/entities/skill.entity';
 import FindAllPaginateAction from './service/find-all-paginate.action';
+import { LanguageProfile } from './entities/language-profile.entity';
+import { Language } from '../language/entities/language.entity';
+import { LanguageModule } from '../language/language.module';
+import LanguagueProfileService from './service/languagueProfileService';
```

Comment:
Creo que el estandar de nombres que seguimos es asi no?
```suggestion
import LanguagueProfileService from './service/languague-profile.service';
```